source/portable/posix/core_pkcs11_pal.c

```diff
@@ -225,6 +243,18 @@ static CK_RV prvHandleToFilename( CK_OBJECT_HANDLE xHandle,
                 *pIsPrivate = ( CK_BBOOL ) CK_TRUE;
                 break;
 
+            case eAwsClaimCertificate:
+                *pcFileName = pkcs11palFILE_NAME_CLAIM_CERTIFICATE;
```

Comment:
There probably should be systems tests added to verify the support of the new labels. Client certificate can be tested by retrieving it from PKCS11 whereas private key can be tested by validating that a signing operation is performed as expected